src/test/url-handling.test.js

```diff
@@ -178,6 +183,20 @@ describe('url upgrading', function() {
       expect(urlStateReducers.getSelectedTab(getState())).toBe('markers-table');
     });
   });
+
+  describe('general checks', function() {
```

Comment:
I'd prefer to have tests named so they read nicely with they are joined. This one would read "general checks won't run if the version is specified". I'd prefer something along the lines of "URL upgrades won't run if the version is specified".

Reply:
I see what you mean, I'll find out something :)